ballista/rust/core/proto/datafusion.proto

```diff
@@ -464,11 +468,17 @@ enum AggregateFunction {
   APPROX_MEDIAN=15;
   APPROX_PERCENTILE_CONT_WITH_WEIGHT = 16;
   GROUPING = 17;
+  MEDIAN=18;
+
+  KYLIN_BITMAP_COUNT_DISTINCT=101;
+  KYLIN_APPROX_PERCENTILE=102;
+  KYLIN_APPROX_COUNT_DISTINCT=103;
```

Comment:
What are these? These don't exist in the DataFusion version of this file

Reply:
Sorry have delete it.

Reply:
They are self define udaf read from pre-aggregate data😂